drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -1,4 +1,4 @@
-# -*- coding: utf-8 -*-
+# -*- coding: utf-8 -*- # pylint: disable=C0302
```

Comment:
I'm not refactoring the drag and drop block class in this ticket, but I will use the human-readable identifier.